### ISSUE TITLE
Support for nested structures and unions

### ISSUE THREAD

**Issue by XVilka** [rizinorg/rizin#1064]
Current RzType doesn't work well with nested types, see `librz/type/type.c` for `FIXME:` nested comments.

Should be started only after https://github.com/rizinorg/rizin/pull/1047 is merged.

**Comment by XVilka** [comment]
They are supported just fine now. If some particular case doesn't work - please open a new issue.
